buildscripts/xds-k8s/test-client.Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build runtime image.
-FROM openjdk:11.0.9.1-jdk
+FROM eclipse-temurin:11.0.19_7-jre
```

Comment:
Even with the apt-get update, we're going to need to regularly update the version here on all branches. What is the hoped for process for that?

Reply:
Yes, makes sense. You're right.